.env.example

```
@@ -1,41 +1,92 @@
+
+# ============================================================================
+# 🤖 AI / ML INTEGRATIONS
+# ============================================================================
+OPENAI_API_KEY="your_openai_api_key"
+HUGGINGFACE_API_TOKEN="your_huggingface_api_token"
+MLFLOW_TRACKING_URI="http://localhost:5000"
+
+# ============================================================================
+# ⚙️ CI/CD SPECIFIC (GitHub Actions, Jenkins, GitLab CI)
+# ============================================================================
+CI_COMMIT_SHA="${{ github.sha }}"
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The CI_COMMIT_SHA key is not assigned properly

(SubstitutionKey)